js/index.js

```diff
@@ -0,0 +1,60 @@
+import data from './info.js';
+
+// const cardList = document.querySelector('.cards-list');
+
```

Comment:
- [ ] Kindly make sure you remove all the commented and unused code lines from your code base to make it more professional and for best practices reasons.